main.py

```diff
@@ -99,6 +99,71 @@ def download_file(
     return response
 
 
+@app.get("/list_bees/")
+def list_bees():
+    """List all objects in the S3 bucket. Hopefully its a bunch of bees in hats!
+    Returns a dictionary where the value of the message key is the list of object
+    names as strings.
+    """
+    response = list_objects()
+
+    return {"message": response}
+
+
+def list_objects() -> list:
+    """Set up access to the S3 bucket and call the generator to create a list of
+    all objects.
+    :return: a list of objects from the bucket
+    """
+
+    s3_bucket = os.environ["AWS_S3_BUCKET"]
```

Comment:
we can centralize this in a class constructor to deduplicate code, but that's for later